plugin/ochttp/httptrace.go

```diff
@@ -169,7 +174,7 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	span.SetAttributes(requestAttrs(r)...)
 
 	r = r.WithContext(ctx)
-	h.handler.ServeHTTP(w, r)
+	h.Handler.ServeHTTP(w, r)
```

Comment:
Should we make the zero value useful by having the default behavior be to call http.DefaultMux with defaultFormat propagation? If we do this, then I think we should also rename or remove NewHandler to address @odeke-em's concern.